src/main/scala/scalaz/stream/writer.scala

```diff
@@ -0,0 +1,20 @@
+package  scalaz.stream
+
+import scalaz.\/._
+
+
+object writer {
+
+  /** Promote a `Process` to a `Writer` that writes nothing. */
+  def lift[F[_], A](p: Process[F, A]): Writer[F, Nothing, A] =
```

Comment:
> @pchlupacek I actually think it would be better to rename this lift to liftW and then to add in an analogous liftO, returning Process[F, W, Nothing].

+1